planner_cspace/src/planner_3d.cpp

```diff
@@ -1249,6 +1258,17 @@ class Planner3d
         {
           if (escaping_)
             status_.error = planner_cspace::PlannerStatus::PATH_NOT_FOUND;
+          else if (max_retry_num_ != -1 && cnt_stuck_ > max_retry_num_)
+          {
+            status_.error = planner_cspace::PlannerStatus::PATH_NOT_FOUND;
+            status_.status = planner_cspace::PlannerStatus::DONE;
+            has_goal_ = false;
+            act_->setAborted(move_base_msgs::MoveBaseResult(),
+                             "Goal is in Rock");
+            ROS_ERROR("Exceeded max_retry_num:%d", max_retry_num_);
+            continue;
+          }
+
```

Comment:
It seems better to remove this empty line and add `{}` to `if (escaping_)` and `else` blocks.

Reply:
Done